packages/cli-platform-apple/src/commands/runCommand/runOptions.ts

```diff
@@ -0,0 +1,46 @@
+import {getDefaultUserTerminal} from '@react-native-community/cli-tools';
+
+export const runOptions = [
```

Comment:
`runOptions` should also contain `buildOptions`

Reply:
cc @thymikee from the beginning `runOptions` contained `buildOptions`, should we split into independent arrays? and then concat them when creating command? 

https://github.com/react-native-community/cli/blob/6143e2935742746b7ccde13160cd2ed2c35c509a/packages/cli-platform-ios/src/commands/runIOS/index.ts#L33

I'm not sure if we want to do it, because this can create a breaking change, when someone just adds `runOptions` 🤔

Reply:
For the sake of this PR I'd keep it as is. But it's a fair call that when extening the `run-ios`, we'd need to document that you also need to import `buildOptions`, which is unexpected